lib/rules/require-spaces-in-for-statement.js

```diff
@@ -58,22 +58,19 @@ module.exports.prototype = {
 
     check: function(file, errors) {
         file.iterateNodesByType('ForStatement', function(node) {
-            var prevToken;
             if (node.test) {
-                var testToken = file.getTokenByRangeStart(node.test.range[0]);
-                prevToken = file.getPrevToken(testToken);
+                var testToken = file.getFirstNodeToken(node.test);
                 errors.assert.whitespaceBetween({
-                    token: prevToken,
+                    token: file.getPrevToken(testToken),
```

Comment:
@hzoo I've thought about it, but better no ;-) We did this: [getCommentAfterToken](https://github.com/jscs-dev/node-jscs/search?utf8=%E2%9C%93&q=getCommentAfterToken&type=Code) — helpers to get all comments before and after token.

→ #987